### PR TITLE
Some alternative words or phrases

### PR DESCRIPTION
**Pull request by spiffxp**
This avoids touching prow in favor of later PR's, since some changes
there may require deprecation/migration etc.

/wg naming

### CODE REVIEW

File: linkcheck/links.go [hunk 1]
```diff
@@ -39,20 +39,20 @@ import (
...
-	regWhiteList = []*regexp.Regexp{
+	exemptRegexes = []*regexp.Regexp{
 		regexp.MustCompile(`https://kubernetes-site\.appspot\.com`),
 		// skip url that doesn't start with an English alphabet, e.g., URLs with IP addresses.
 		regexp.MustCompile(`https?://[^A-Za-z].*`),
 		regexp.MustCompile(`https?://localhost.*`),
 	}
-	// URLs listed in the fullURLWhiteList won't be checked. This separated from
-	// the RegWhiteList to improve efficiency. This list includes dummy URLs that
+	// URLs listed in exemptURLs won't be checked. This is seperate from
```
Comment (MushuEE) [inline linkcheck/links.go:52]: nit: separate

File: experiment/migrate_testgrid_tabs.py [hunk 2]
```diff
@@ -160,13 +160,13 @@ def move_group(config, group, prow_dir):
 def assert_tab_keys(tab):
     """Asserts if a dashboard tab is able to be migrated.
 
-    To be migratable, the annotations must only contain whitelisted keys
+    To be migratable, the annotations must only contain allowed keys
     AND alert_options, if present, must contain and only contain "alert_mail_to_addresses"
     """
-    whitelist = ["name", "description", "test_group_name", "alert_options",
+    allowedKeys = ["name", "description", "test_group_name", "alert_options",
                  "num_failures_to_alert", "alert_stale_results_hours", "num_columns_recent"]
```
Comment (MushuEE) [inline experiment/migrate_testgrid_tabs.py:167]: Also seems like pylint wants quotes aligned (2 spaces) here and 185

**Comment by BenTheElder** [comment]
/approve
(lgtm once above comments are fixed)

File: experiment/migrate_testgrid_tabs.py [hunk 3]
```diff
@@ -179,12 +179,12 @@ def assert_tab_keys(tab):
 def assert_group_keys(group):
     """Asserts if a testgroup is able to be migrated.
 
-    To be migratable, the group must only contain whitelisted keys
+    To be migratable, the group must only contain allowed keys
     """
-    whitelist = ["name", "gcs_prefix", "alert_stale_results_hours",
+    allowedKeys = ["name", "gcs_prefix", "alert_stale_results_hours",
                  "num_failures_to_alert", "num_columns_recent"]
```
Comment (MushuEE) [inline experiment/migrate_testgrid_tabs.py:185]:
```suggestion
                   "num_failures_to_alert", "num_columns_recent"]
```

Comment (MushuEE) [inline experiment/migrate_testgrid_tabs.py:185]:
```suggestion
    allowedKeys = ["name", "gcs_prefix", "alert_stale_results_hours",
                   "num_failures_to_alert", "num_columns_recent"]
```

File: experiment/migrate_testgrid_tabs.py (hunk 2 above)
Comment (MushuEE) [inline experiment/migrate_testgrid_tabs.py:167]:
```suggestion
    allowedKeys = ["name", "description", "test_group_name", "alert_options",
                   "num_failures_to_alert", "alert_stale_results_hours", "num_columns_recent"]
```

**Comment by spiffxp** [comment]
Squashed suggestion commits locally and rebased, next time I'll try using the "add suggestion to batch" button

**Comment by spiffxp** [comment]
/test pull-test-infra-bazel
good ol' `read: connection reset by peer`
```
ERROR: An error occurred during the fetch of repository 'gcloud-go':
   Pull command failed: 2020/08/27 04:25:59 Running the Image Puller to pull images from a Docker Registry...
2020/08/27 04:26:00 Image pull was unsuccessful: unable to save remote image gcr.io/k8s-testimages/gcloud-in-go@sha256:ca9873d744f19e77ceafb499846248a82cde74ab5a613dd394348e7904d08d71: unable to write image layers: unable to write image layer: unable to write the contents of layer 0 to /bazel-scratch/.cache/bazel/_bazel_root/05618a594cb3499a4b912817865867cd/external/gcloud-go/image/000.tar.gz: read tcp 10.60.137.106:53248->172.217.212.128:443: read: connection reset by peer
```

**Pull request merged**